controllers/keystoneapi_controller.go

```diff
@@ -1203,6 +1207,21 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 			endptConfig["SSLCertificateFile"] = fmt.Sprintf("/etc/pki/tls/certs/%s.crt", endpt.String())
 			endptConfig["SSLCertificateKeyFile"] = fmt.Sprintf("/etc/pki/tls/private/%s.key", endpt.String())
 		}
+		if instance.Spec.OIDCFederation.EnableFederation {
+			endptConfig["EnableFederation"] = true
+			endptConfig["OIDCClaimPrefix"] = instance.Spec.OIDCFederation.OIDCClaimPrefix
+			endptConfig["OIDCResponseType"] = instance.Spec.OIDCFederation.OIDCClaimPrefix
+			endptConfig["OIDCScope"] = instance.Spec.OIDCFederation.OIDCScope
+			endptConfig["OIDCProviderMetadataUrl"] = instance.Spec.OIDCFederation.OIDCProviderMetadataURL
+			endptConfig["OIDCClientID"] = instance.Spec.OIDCFederation.OIDCClientID
+			endptConfig["OIDCClientSecret"] = instance.Spec.OIDCFederation.OIDCClientSecret
```

Comment:
This doesn't work like this.  Look at https://github.com/openstack-k8s-operators/keystone-operator/blob/47b983c3ab77e998ebb80b6024a9fdc6a75585fa/api/v1beta1/keystoneapi.go#L137 for an example on how to extract the passwords from osp.secret